src/components/organisms/NavBarSchedule/NavBarSchedule.tsx

```diff
@@ -20,15 +20,14 @@ import {
 } from "utils/event";
 import { range } from "utils/range";
 import { formatDateRelativeToNow } from "utils/time";
-import { getLastUrlParam, getUrlWithoutTrailingSlash } from "utils/url";
 
 import { useRelatedVenues } from "hooks/useRelatedVenues";
 import { useRoomRecentUsersList } from "hooks/useRoomRecentUsersList";
 import { useShowHide } from "hooks/useShowHide";
 import { useUser } from "hooks/useUser";
 import useVenueScheduleEvents from "hooks/useVenueScheduleEvents";
 
-import { NavBarScheduleWeather } from "components/molecules/NavBarScheduleWeather";
+// import { NavBarScheduleWeather } from "components/molecules/NavBarScheduleWeather";
```

Comment:
Comments in the import section can cause issues with the automatic sorting plugin

```suggestion
```